### PR TITLE
 .moban.yaml: Add CI check scripts

### PR DESCRIPTION
**Pull request by jayvdb**
Related to https://gitlab.com/coala/mobans/issues/65

### CODE REVIEW

**Comment by jayvdb** [comment]
Depends on https://github.com/coala/coala/pull/5634

File: .moban.yaml [hunk 1]
```diff
@@ -59,3 +59,6 @@ targets:
   - netlify.toml: docs/netlify.toml
   - .misc/appveyor.yml: ci/appveyor.yml.jj2
   - .misc/run_with_env.cmd: run_with_env.cmd
+  - .misc/check_moban.sh: ci/check_moban.sh
+  - .misc/check_setuptools.py: ci/check_setuptools.py
+  - .misc/check_unsupported.sh: ci/check_unsupported.sh
```
Comment (li-boxuan) [inline .moban.yaml]: `ci/check_unsupported.sh.jj2`? CI failing https://travis-ci.org/coala/coala/jobs/406112373#L524

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5635/commits/efb5841c30b1672f18dd92e4cee280fbe4930368 https://github.com/coala/coala/pull/5635/commits/9d674b5b66433681557a794c8706d8129b205c3e

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**